tests/installation/partitioning_lvm.pm

```diff
@@ -17,23 +17,39 @@ sub run() {
     my $self = shift;
 
     send_key "alt-d";
-    sleep 2;
 
-    send_key "alt-l", 1;    # enable LVM-based proposal
-    if (get_var("ENCRYPT")) {
-        send_key "alt-y", 1;
-        assert_screen "inst-encrypt-password-prompt";
-        type_password;
-        send_key "tab";
-        type_password;
-        send_key "ret", 1;
-        assert_screen "partition-cryptlvm-summary";
+    if (check_screen('inst-partition-radio-buttons', 10)) {    # detect whether new (Radio Buttons) YaST behaviour
+        if (get_var("ENCRYPT")) {
+            send_key 'alt-e';
+            send_key 'alt-o';
```

Comment:
With the new workflow - no, it's the only way of getting it to prompt for the password